packages/params/src/index.ts

```diff
@@ -47,7 +47,7 @@ export const {
   MAX_EFFECTIVE_BALANCE,
   EFFECTIVE_BALANCE_INCREMENT,
   MIN_ATTESTATION_INCLUSION_DELAY,
-  SLOTS_PER_EPOCH,
+  SLOTS_PER_EPOCH, // <spec preset_var="SLOTS_PER_EPOCH" fork="deneb" style="hash" hash="cb41af43" />
```

Comment:
Any reason you tag `SLOT_PER_EPOCH` here and not https://github.com/ChainSafe/lodestar/blob/2b5494153805e62342d612c56fb508282c16fa50/packages/params/src/presets/mainnet.ts#L38 like you did for other presets?